clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -688,7 +688,8 @@ public void testOffsetFetchRequestTimeoutRequests(final Errors error) {
     }
 
     private boolean isRetriableOnOffsetFetch(Errors error) {
-        return error == Errors.NOT_COORDINATOR || error == Errors.COORDINATOR_LOAD_IN_PROGRESS || error == Errors.COORDINATOR_NOT_AVAILABLE;
+        return error == Errors.NOT_COORDINATOR || error == Errors.COORDINATOR_LOAD_IN_PROGRESS || error == Errors.COORDINATOR_NOT_AVAILABLE
+                || error == Errors.UNKNOWN_TOPIC_OR_PARTITION || error == Errors.REQUEST_TIMED_OUT || error == Errors.UNSTABLE_OFFSET_COMMIT;
```

Comment:
Hi @lianetm,
Thanks for pointing that out. You are right, in this case, we should use `instanceof RetriableException` instead of the helper method, as we already consider all retriable exceptions should be retried. I will remove the method and replace its usage with the instanceof expression. 
Thanks.